reinforcement_learning/rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb

```diff
@@ -115,7 +115,10 @@
    "cell_type": "code",
```

Comment:
<p>Minor: what is the logic that we need to separate this our as a single cell, but no the instance_type</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1580/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>The papermill website has a good explanation for this: https://papermill.readthedocs.io/en/latest/usage-parameterize.html#how-parameters-work</p>

Reply:
Correct me if I am wrong, it looks like we are parameterize both "train_instance_count" and "instance_type". Shouldn't we add "instance_type" to the parameterized cell as well?

Reply:
> The papermill website has a good explanation for this: https://papermill.readthedocs.io/en/latest/usage-parameterize.html#how-parameters-work

It looks like papermill has different interfaces for jupyter notebook and jupyter lab, do we have a sanity check whether the changes brought by papermill will not break anything down on the notebook console?

Reply:
Yeah, we have the parameters for instance_type as well.
```
"tags": [
     "parameters"
    ]
   },
   "outputs": [],
   "source": [
    "# run in local mode?\n",
    "local_mode = False\n",
    "\n",
    "if local_mode:\n",
    "    instance_type = 'local'\n",
    "else:\n",
    "    instance_type = \"ml.m4.4xlarge\""
   ]
```

Reply:
`It looks like papermill has different interfaces for jupyter notebook and jupyter lab, do we have a sanity check whether the changes brought by papermill will not break anything down on the notebook console?`

I checked the UX on JupyterLab as well, there is no difference with the parameters tag added to the cells. Here's a screenshot:
<img width="881" alt="Screen Shot 2020-10-06 at 12 59 41 PM" src="https://user-images.githubusercontent.com/6972804/95253861-e1a34680-07d3-11eb-8014-541c386f499d.png">
<img width="1189" alt="Screen Shot 2020-10-06 at 12 59 50 PM" src="https://user-images.githubusercontent.com/6972804/95253875-e7009100-07d3-11eb-9873-60c96746b282.png">

Reply:
> Yeah, we have the parameters for instance_type as well.
> 
> ```
> "tags": [
>      "parameters"
>     ]
>    },
>    "outputs": [],
>    "source": [
>     "# run in local mode?\n",
>     "local_mode = False\n",
>     "\n",
>     "if local_mode:\n",
>     "    instance_type = 'local'\n",
>     "else:\n",
>     "    instance_type = \"ml.m4.4xlarge\""
>    ]
> ```

In this case, can 'local_mode' be one of the parameters?

Reply:
Added